From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: bashbug-editor

# DP: send bug reports to Debian bash maintainer too.

# DP: send bug reports to Debian bash maintainer too.
---
 support/bashbug.sh.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/support/bashbug.sh.in b/support/bashbug.sh.in
index 4f558a5..65b7a2f 100644
--- a/support/bashbug.sh.in
+++ b/support/bashbug.sh.in
@@ -117,6 +117,7 @@ alpha*|beta*|devel*|rc*)
 		esac ;;
 esac
 
+BUGBASH="${BUGBASH},bash@packages.debian.org"
 BUGADDR="${1-$BUGBASH}"
 
 if [ -z "$DEFEDITOR" ] && [ -z "$EDITOR" ]; then
